keeper-gradle-plugin/src/main/java/com/slack/keeper/ZipFlingerExt.kt

```diff
@@ -30,7 +30,7 @@ internal fun File.classesSequence(): Sequence<Pair<String, File>> {
     .filter { it.extension == "class" }
     .filterNot { "META-INF" in it.name }
     .sortedBy { it.invariantSeparatorsPath }
-    .map { it.absolutePath.removePrefix(prefix).removePrefix("/") to it }
+    .map { it.absolutePath.removePrefix(prefix).removePrefix(File.separator).replace(File.separator, "/") to it }
```

Comment:
Is the replace correct? Also are you willing to set up a CI matrix for windows coverage?

Reply:
I think the replace becomes redundant on Linux. It's not nonfunctional because of this, just.. ugly, maybe?

And, yeah, why not. I'll try to get a workflow done this week, maybe see if Mike could help.

Reply:
🪄  here is a workflow that exercises windows and shows the problem

- #148

Reply:
(also maybe worth mentioning that on my fork all the existing tests pass, and we have a local build of this keeper jar in use now in Anki-Android with this fix and it's working great - PR to integrate it is just pending a final review)

Reply:
Quick status update - the fix definitely works for the plugin itself but there are two problems with this commit that I discovered when I integrated the commit + my windows workflow to see it in action:

- the commit fails spotless check and needs a `./gradlew :spotlessApply` (easy enough)
- the test harness has some windows-filesystem issues in test project generation so[ it still fails on windows](https://github.com/mikehardy/keeper/actions/runs/9683300818/job/26718408438?pr=2), looking into that, and I'll post again when it's good

Reply:
@ZacSweers got this all sorted out. There is a combo PR up at #149 where everything is tested and working in CI on my fork and should go green if you approve the workflow run over there. #149 should supercede this PR